awx/main/models/credential.py

```diff
@@ -81,6 +81,38 @@ def build_safe_env(env):
     return safe_env
 
 
+def check_gateway_for_user_in_organization(user, organization, requesting_user):
+    from ansible_base.resource_registry.tasks.sync import get_resource_server_client
+    from ansible_base.resource_registry.utils.settings import resource_server_defined
+
+    if not resource_server_defined():
+        return False
+
+    if not requesting_user:
+        return False
+
+    client = get_resource_server_client(settings.RESOURCE_SERVICE_PATH, jwt_user_id=str(requesting_user.resource.ansible_id), raise_if_bad_request=True)
```

Comment:
Is `RESOURCE_SERVICE_PATH` a new setting? Do we need installer changes to add it?

Reply:
it's newish but should have a default assigned